backend/services/soft_deletion.py

```
@@ -48,27 +48,24 @@ def soft_delete_scenario(
             # Store scenario info before deletion
             scenario_title = scenario.title
```

Ruff (0.14.1)

Local variable `scenario_title` is assigned to but never used

Remove assignment to unused variable `scenario_title`

(F841)